datalad_dataverse/remote.py

```diff
@@ -21,6 +21,47 @@
 )
 
 
+LEADING_DOT_REPLACEMENT = "_._"
+
+
+def mangle_directory_names(path):
```

Comment:
I think this function need to do more than just replacing a leading dot. https://github.com/IQSS/dataverse/issues/8807#issuecomment-1164434278 has the full picture.